content/components/relative-time.mdx

```diff
@@ -0,0 +1,115 @@
+
+If the future date is due to happen within the current month, the relative time outputs the format of `in X seconds/minutes/hours/days`. If the time difference is less than 1 minute, it shows `just now`.
+
+If the future date is due to happen within the following months, the relative time outputs the format of `on D MMM` or `on D MMM, YYYY` when following years.
+
+### Micro
+
+The `micro` variant shows a smaller version of the relative time. It is useful for displaying the relative time in a more compact space or in a mobile environment. The output shows the relative time in the format of `Xs/h/d`, `D MMM` after a month, or `D MMM, YYYY` after a year.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/912236/191981619-4b2a5625-aa28-44d4-a650-fcda9f349b6f.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Use micro to display time in a more compact space or in a mobile environment.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/912236/191997424-821afe8a-39d9-47c4-9203-ebaee182f3b1.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Avoid using the micro option when there's a prominent space. Use the default option instead.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Usage
+
+A relative time is intended to be used within a sentence, following the action that it's relative to.
+
+<img
+  width="960"
+  alt=""
+  src="https://user-images.githubusercontent.com/912236/191977760-bccc2197-6ecb-4487-9c5d-0d3732cc7f9e.png"
+/>
+
+### Isolated
+
+A relative time can be used in isolation when the content and the surrounding elements make it clear what the time is relative to.
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <img
+      width="460"
+      alt=""
+      src="https://user-images.githubusercontent.com/912236/191997579-7a5be204-91ed-4163-97fc-18cd6e82a0c7.png"
+    />
+    <Caption>Used in a table where each item title references the action that was performed.</Caption>
+  </div>
+  <div>
+    <img
+      width="460"
+      alt=""
+      src="https://user-images.githubusercontent.com/912236/191965093-80c1858f-5d36-4caf-aecf-fd333c2c104c.png"
+    />
+    <Caption>Used in timeline where the section title references the action.</Caption>
+  </div>
+</Box>
+
+When used in isolation, the default format can change to `micro` when the space is limited.
+
+<CustomVideoPlayer
+  width="100%"
+  loop
+  src="https://user-images.githubusercontent.com/912236/192255353-88749767-f1d1-49af-8145-86b60f5215b8.mp4"
+/>
+<Caption>The micro format is used for responsive layouts.</Caption>
+
+### Links
+
+A relative time is used as plain text in most cases. However, it can be used as an anchor link to point to an element within a view that contains relevant or nested information.
+
+<img
```

Comment:
The link on hover should have an underline.

Reply:
You are right 👍, fixed!